schemas/2019-03-01-hybrid/deploymentTemplate.json

```diff
@@ -0,0 +1,3639 @@
+﻿{
+  "id": "https://raw.githubusercontent.com/t-anba/azure-resource-manager-schemas/t-anba-AzsDeploymentTemplate-final/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Make sure you replace this with the correct URL - I think in this case it should be `https://schema.management.azure.com/schemas/2019-03-01-hybrid/deploymentTemplate.json#`